app/src/main/java/edu/nguyenmy/marveladroid_7/screen/BaseAdapter.java

```diff
@@ -0,0 +1,25 @@
+package edu.nguyenmy.marveladroid_7.screen;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+
+/**
+ * Created by DELL on 11/8/2017.
+ */
+
+public abstract class BaseAdapter<V extends RecyclerView.ViewHolder> extends RecyclerView.Adapter<V> {
+    private final Context mContext;
+
+    protected BaseAdapter(@NonNull Context mContext) {
```

Comment:
check lại convention 
2.1 Java Style Rules

Field names need to be defined at first and follow this conventions.

Non-public, non-static field names start with m.
Static field names start with s.
Other fields start with a lower case letter.
Public static final fields (constants) are ALL_CAPS_WITH_UNDERSCORES.
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md